bindings/wasm/src/account/wasm_account_builder.rs

```diff
@@ -0,0 +1,153 @@
+
+/// An [`Account`] builder for easy account configuration.
+///
+/// To reduce memory usage, accounts created from the same builder share the same `Storage`
+/// used to store identities, and the same {@link Client} used to publish identities to the Tangle.
+///
+/// The configuration on the other hand is cloned, and therefore unique for each built account.
+/// This means a builder can be reconfigured in-between account creations, without affecting
+/// the configuration of previously built accounts.
+#[wasm_bindgen(js_name = AccountBuilder)]
+pub struct WasmAccountBuilder(Rc<RefCell<AccountBuilder>>);
+
+#[wasm_bindgen(js_class = AccountBuilder)]
+impl WasmAccountBuilder {
+  /// Creates a new `AccountBuilder`.
+  #[wasm_bindgen(constructor)]
+  pub fn new(options: Option<AccountBuilderOptions>) -> Result<WasmAccountBuilder> {
+    let default_config: AccountConfig = AccountConfig::default();
+    let mut builder = AccountBuilder::new();
+
+    if let Some(builder_options) = options {
+      builder = builder
+        .autopublish(builder_options.autopublish().unwrap_or(default_config.autopublish))
+        .autosave(
+          builder_options
+            .autosave()
+            .map(|auto_save| auto_save.0)
+            .unwrap_or(default_config.autosave),
+        );
+      //todo storage
+      if let Some(mut config) = builder_options.clientConfig() {
+        let client: WasmClient = WasmClient::from_config(&mut config)?;
+        builder = builder.client(Arc::new(client.client.as_ref().clone()));
+      };
+    }
+
+    Ok(Self(Rc::new(RefCell::new(builder))))
+  }
+
+  /// Loads an existing identity with the specified `did` using the current builder configuration.
+  /// The identity must exist in the configured `Storage`.
+  #[wasm_bindgen(js_name = loadIdentity)]
+  pub fn load_identity(&mut self, did: &WasmDID) -> Result<PromiseAccount> {
+    let builder = self.0.clone();
+    let did = did.clone();
+    let promise: Promise = future_to_promise(async move {
+      builder
+        .as_ref()
+        .borrow_mut()
+        .load_identity(did.0)
+        .await
+        .map(WasmAccount::from)
+        .map(Into::into)
+        .wasm_result()
+    });
+    Ok(promise.unchecked_into::<PromiseAccount>())
+  }
+
+  /// Creates a new identity based on the builder configuration and returns
+  /// an {@link Account} object to manage it.
+  ///
+  /// The identity is stored locally in the `Storage`. The DID network is automatically determined
+  /// by the {@link Client} used to publish it.
+  ///
+  /// @See {@link IdentitySetup} to customize the identity creation.
+  #[wasm_bindgen(js_name = createIdentity)]
+  pub fn create_identity(&mut self, identity_setup: Option<WasmIdentitySetup>) -> Result<PromiseAccount> {
+    // Create IdentitySetup
+    let setup: IdentitySetup = identity_setup.map(IdentitySetup::from).unwrap_or_default();
+
+    // Call the builder.
+    let builder: Rc<RefCell<AccountBuilder>> = self.0.clone();
```

Comment:
Nit: the comments here are unnecessary in my opinion.

```suggestion
    let setup: IdentitySetup = identity_setup.map(IdentitySetup::from).unwrap_or_default();

    let builder: Rc<RefCell<AccountBuilder>> = self.0.clone();
```